cfme/services/myservice/ssui.py

```diff
@@ -216,6 +249,46 @@ def launch_vm_console(self, catalog_item):
     return vm_obj
 
 
+@MiqImplementationContext.external_for(MyService.view_snapshots, ViaSSUI)
+def view_snapshots(self):
+    view = navigate_to(self, 'ViewSnapshot')
```

Comment:
We have to wait_for_view=True here
view = navigate_to(self, 'ViewSnapshot', wait_for_view=True)

Then the wait_for(...)
can be removed.